spec/features/favorites/delete_spec.rb

```diff
@@ -26,4 +24,32 @@
     expect(page).to_not have_xpath("//img[@src='#{pet1.image}']")
 
   end
+  
+  it "Pet show pages will give a 'Remove All Pets From Favorites'" do
+    shelter = Shelter.create(name: "Primary Shelter", address: "123 Maple Ave.", city: "Denver", state: "CO", zip: "80438")
+    pet1 = Pet.create(name: "Bonnie", image: "http://www.gsgsrescue.org/assets/files/dogs/2020/06/IMG_1639_1.jpg", approximate_age: "13", sex: "Female", shelter_id: shelter.id, status: "Adoptable")
+
+    visit "/pets/#{pet1.id}"
+    
+    expect(page).to have_content('Favorite')
+    
+    click_button 'Favorite'
+    
+    within("navbar") do
+      click_link "Favorites - 1"
+    end
+    
+    expect(current_path).to eq("/favorites")
+    expect(page).to have_content(pet1.name)
+    expect(page).to have_xpath("//img[@src='#{pet1.image}']")
+    expect(page).to have_button('Remove All Pets From Favorites')
```

Comment:
since line 46 says to click the button with that text, I don't think this expectation is necessary